### PR TITLE
Create WIX files for building Windows Installer

### PR DESCRIPTION
**Pull request by arastooz**

### CODE REVIEW

**Comment by jacobmarble** [comment]
LICENSE.rtf can be removed, unless you have a particular reason to add it.

File: wix/cups-windows/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Windows Installer
```
Comment (jacobmarble) [inline wix/cups-windows/README.md]: Why is this file in wix/cups-windows? Any objection to just wix?

File: wix/cups-windows/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Build Requirements
+The WIX toolset is required to build the Windows Installer file, the WIX toolset is required. 
+It can be downloaded from http://wixtoolset.org.
+
+## Build Instructions
+Run candle.exe to build wixobj file from the wxs file:
+candle.exe -arch x64 cups-connector.wxs
```
Comment (jacobmarble) [inline wix/cups-windows/README.md]: The Windows version of the CUPS Connector is "Windows Connector" or "Cloud Print Windows Connector". Please replace all instances of "cups" or "CUPS" with "windows" or "Windows".

**Comment by jacobmarble** [comment]
Since this ends up being a small change, please git rebase to reduce to one commit.

This is a significant moment in the Kingdom of Connector. Thanks a million. I'll test it out tomorrow.

**Comment by jacobmarble** [comment]
Never mind the LICENSE.rtf comment. I see that you mentioned it's required for Windows Installer. Please place it with the other wix-specific files.

**Comment by arastooz** [comment]
Done.

**Comment by jacobmarble** [comment]
@arastooz Thank you.

**Pull request merged**